### DISCUSSION TITLE
CSS doesnt apply in pages

### DISCUSSION THREAD

**Discussion by kkiravi** [community/community#138992] [New to GitHub] [answered]
Hello, this is the first repository that I made, I tried to host a website and HTML was working well, however, CSS doesn't apply to my website. I tried searching for that problem on the internet, but it didn't help me. what do I do? Thank you for the answers 
https://github.com/kkiravi/disabilitiesservice

**Comment by davevad93** [comment, marked as answer]
Hi @kkiravi ,

The issue is with your file paths in both your `html` and `css`, for example `/public/...` , just remove the forward slash at the beginning of your paths

<img width="257" alt="2" src="https://github.com/user-attachments/assets/e42dc7a6-3c64-4dc0-93e1-c594306f3c05">
<br>
<br>

I guess that's how your website should look like:

<img width="1259" alt="1" src="https://github.com/user-attachments/assets/4613bfe5-9013-4078-8180-b815f7a95c9e">

Do the same for all your images and script paths in the `html`.

**Reply by kkiravi** [reply]
it worked!! thank you very much!! you saved my day!!

**Reply by davevad93** [reply]
You're welcome @kkiravi , have a nice day! 👍
